.changeset/odd-badgers-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-shell': minor
+---
+
+feat(menu): add new icon option for side nav menu
```

Comment:
you have to upgrade UI-kit unless the version is already correct

Reply:
I'm waiting on https://github.com/commercetools/ui-kit/pull/1790 to be approved and merged, since I need the patch for the icons.

Reply:
aha. very well

Reply:
i added `ship it`now.

have fun

Reply:
Thank you Adnan! 🙏